### PR TITLE
✨ Add Grafana Plugin migration in alpha generate subcommand -- Step 1

### PR DESCRIPTION
**Pull request by yyy1000**
### Description:

Support Grafana Plugin migration in the alpha generate subcommand. 

This is the first step, which will only call 'kubebuilder' command.

### CODE REVIEW

File: pkg/rescaffold/migrate.go [hunk 1]
```diff
@@ -142,12 +152,30 @@ func kubebuilderCreate(store store.Store) error {
...
+	// If the plugin is already in the plugin chain, we don't need call 'edit' method
+	// Because the plugin is already migrated in the previous step
+	plugins := store.Config().GetPluginChain()
+	if slices.Contains(plugins, grafanaPluginKey) {
+		return nil
+	}
+	// If the plugin is not in the plugin chain, we need to call 'edit' method to add the plugin
+	var grafanaPlugin struct{}
+	err := store.Config().DecodePluginConfig(grafanaPluginKey, grafanaPlugin)
+	if err == nil {
```
Comment (Kavinjsir) [inline pkg/rescaffold/migrate.go:165]: Nits: I think you can drop `else` here for better readability, for instance:

```go
	if errors.As(err, &config.PluginKeyNotFoundError{}) {
		return nil
	}

	if err != nil {
		return fmt.Errorf("Failed to Decode Grafana Plugin: %s. %v", grafanaPluginKey, err)
	}

	return migrateGrafanaPlugin()
```

**Review by Kavinjsir — APPROVED** [review]

File: pkg/rescaffold/migrate.go (hunk 1 above)
Comment (yyy1000) [reply pkg/rescaffold/migrate.go]: Good idea!

**Review by camilamacedo86 — APPROVED** [review]
/lgtm

**Pull request merged**